tcms/testruns/tests/test_permissions.py

```diff
@@ -128,3 +128,37 @@ def verify_post_with_permission(self):
             self.assertEqual(case.history.latest().history_id, case_run.case_text_version)
             self.assertEqual(last_run.build, case_run.build)
             self.assertEqual(None, case_run.close_date)
+
+
+class MenuAddCommentItemTestCase(PermissionsTestCase):
+    permission_label = 'django_comments.add_comment'
+    http_method_names = ['get']
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.test_run = factories.TestRunFactory()
+
+        cls.url = reverse('testruns-get', args=[cls.test_run.pk])
+        cls.post_data={'summary': 'mock',
+                                }
```

Comment:
The above 2 lines cause flake8 failures. Fix them and next time run flake8 locally before pushing.